WORKING-GROUPS.md

```diff
@@ -27,6 +27,7 @@ The current working groups are:
 | Networking | [Folder](https://drive.google.com/corp/drive/u/0/folders/0BzW5bSyKst8Jb1QwTXl1WTRqWHM) | [Forum](https://discuss.istio.io/c/networking) | [#networking](https://istio.slack.com/messages/C38CF1PEC/) | [Notes](https://docs.google.com/document/d/1xHy2jQ8oiwMponMVY2zJr2eUAmHW_Hi9JK42a7cg5Pc/edit#heading=h.2ju2wl4o5jbc) | [Hangouts Meet](https://meet.google.com/xjj-ujhi-qfk?hs=122) | [YouTube](https://www.youtube.com/playlist?list=PL7wB27eZmdffpmIKb5tVthiiQLSXenCK-) | Traffic Management, TCP Support, Additional L7 protocols, Proxy injection
 | Performance and Scalability | [Folder](https://drive.google.com/corp/drive/u/0/folders/1Zpi5TcBPSqGno96WAq5QkcNKi16opaTL) | [Forum](https://discuss.istio.io/c/performance-and-scalability) | [#perf-and-scalability](https://istio.slack.com/messages/C7S85G0SY/) | [Notes](https://goo.gl/ENFQWb) | [Hangouts Meet](https://meet.google.com/njo-sthe-fqv) | n/a | Performance and scalability characterization and improvements
 | Policies and Telemetry | [Folder](https://drive.google.com/corp/drive/u/0/folders/0BzW5bSyKst8JbWtHOEo1STc1dGM) | [Forum](https://discuss.istio.io/c/policies-and-telemetry) | [#policies-telemetry](https://istio.slack.com/messages/C382V8Q92/) | [Notes](https://docs.google.com/document/d/1pn9QdRcoyT_nxOwzklsiYpt7OQraaSDfmtN14XTOrN0/edit) | [Hangouts Meet](https://meet.google.com/cmo-jgug-uhj) | [YouTube](https://www.youtube.com/playlist?list=PL7wB27eZmdffF-9nyaw01Ni_0GOWBzaF4) | Istio extensions for features such as Rate Limiting, Tracing, Monitoring, Logging
+| Product Security | [Folder](https://drive.google.com/corp/drive/u/0/folders/1-KQmAKK4LrSe7NiJK8dwGHD9a7zw_PYM) | [Report a vulnerability](https://istio.io/about/security-vulnerabilities/) |  | [Notes](https://docs.google.com/document/d/1fGi8G9wGfJhA23CT9Awp2bBdasanILh3gK9ib5hrTlE/edit#) | [Hangouts Meet](https://meet.google.com/vao-otzc-hvx) | | Product Security: Vulnerability, security guidelines, threats
```

Comment:
Should the leads for this new group be posted in the table below? At least I thought they were determined and could be added. Maybe I'm being premature.

Reply:
added